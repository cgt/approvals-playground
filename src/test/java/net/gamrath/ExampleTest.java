package net.gamrath;

import org.approvaltests.Approvals;
import org.approvaltests.JsonJacksonApprovals;
import org.junit.jupiter.api.Test;

class ExampleTest {
    @Test
    void myApprovalTestWithJSON() {
        JsonJacksonApprovals.verifyAsJson(new Person("John", 43));
    }

    @Test
    void myApprovalTest() {
        Approvals.verify(new Person("Alice", 10));
    }

    record Person(String name, int age) {
    }
}