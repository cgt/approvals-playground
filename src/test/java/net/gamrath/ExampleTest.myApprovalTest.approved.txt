Person[name=Alice, age=10]